### PR TITLE
18_jan_billboard python solution added & linked

### PR DESCRIPTION
**Pull request by cskitty**

### CODE REVIEW

File: solutions/USACO jan18-billboard.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+[Official Analysis](http://www.usaco.org/current/data/sol_billboard_bronze_jan18.html)
+
+<LanguageSection>
+<JavaSection>
+See official editorial: http://www.usaco.org/current/data/sol_lemonade_silver_open18.html
+</JavaSection>
+
+<PySection>
+
+## C++ Implementation
```
Comment (thecodingwizard) [inline solutions/USACO jan18-billboard.mdx:17]: Do you mean python impl?

Comment (cskitty) [reply solutions/USACO jan18-billboard.mdx]: Fixed it

File: solutions/USACO jan18-billboard.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+source: USACO Bronze 2018 January
+title: Blocked Billboard II
+author: Melody Yu
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_billboard_bronze_jan18.html)
+
+<LanguageSection>
+<JavaSection>
+See official editorial: http://www.usaco.org/current/data/sol_lemonade_silver_open18.html
```
Comment (bqi343) [inline solutions/USACO jan18-billboard.mdx:12]: huh

File: solutions/USACO jan18-billboard.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+</JavaSection>
+
+<PySection>
+
+## python Implementation
```
Comment (dolphingarlic) [inline solutions/USACO jan18-billboard.mdx:17]:
```suggestion
## Python Implementation
```

File: solutions/USACO jan18-billboard.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+```py
+fIn, fOut = open("billboard.in"), open("billboard.out", "w")
+
+x1, y1, x2, y2 = tuple(map(lambda x: int(x), fIn.readline().split()))
+x3, y3, x4, y4 = tuple(map(lambda x: int(x), fIn.readline().split()))
```
Comment (dolphingarlic) [inline solutions/USACO jan18-billboard.mdx:23]: You don't need to convert the map into a tuple ;)

File: solutions/USACO jan18-billboard.mdx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    tlCorner = True
+
+if x4 >= x2 and y3 <= y1:
+    brCorner = True
+
+
+cornerCount = len(list(filter(lambda x:x, [blCorner, trCorner, brCorner, tlCorner])))
+
+if blCorner and trCorner:
+    fOut.write(str(0))
```
Comment (dolphingarlic) [inline solutions/USACO jan18-billboard.mdx:46]:
```suggestion
    fOut.write('0')
```

File: solutions/USACO jan18-billboard.mdx (hunk 2 above)
Comment (cskitty) [reply solutions/USACO jan18-billboard.mdx]: fixed link

File: solutions/USACO jan18-billboard.mdx (hunk 4 above)
Comment (cskitty) [reply solutions/USACO jan18-billboard.mdx]: got it!

File: solutions/USACO jan18-billboard.mdx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+source: USACO Bronze 2018 January
+title: Blocked Billboard II
+author: Melody Yu
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_billboard_bronze_jan18.html)
+
+<LanguageSection>
+<JavaSection>
+See official editorial: http://www.usaco.org/current/data/sol_billboard_bronze_jan18.html
```
Comment (bqi343) [inline solutions/USACO jan18-billboard.mdx:12]: might as well just not include this part

File: solutions/USACO jan18-billboard.mdx [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+</JavaSection>
+
+<PySection>
+
+## Python Implementation
+
+```py
+fIn, fOut = open("billboard.in"), open("billboard.out", "w")
+
+x1, y1, x2, y2 = (map(lambda x: int(x), fIn.readline().split()))
```
Comment (bqi343) [inline solutions/USACO jan18-billboard.mdx:22]: don't need parentheses around map

File: solutions/USACO jan18-billboard.mdx (hunk 6 above)
Comment (cskitty) [reply solutions/USACO jan18-billboard.mdx]: got it

File: solutions/USACO jan18-billboard.mdx (hunk 7 above)
Comment (cskitty) [reply solutions/USACO jan18-billboard.mdx]: fixed!

**Pull request merged**